emcc.py

```diff
@@ -395,6 +395,22 @@ def apply_settings(user_settings):
     if key == 'WASM_OBJECT_FILES':
       settings.LTO = 0 if value else 'full'
 
+  apply_min_browser_versions(user_settings)
+
+
+def default_min_browser_version(user_settings, browser, version):
+  default_setting(user_settings, f"MIN_{browser.upper()}_VERSION", version)
+
+
+# apply minimum browser version defaults based on user settings. if
+# a user requests a feature that we know is only supported in browsers
+# from a specific version and above, we can assume that browser version.
+def apply_min_browser_versions(user_settings):
+  if settings.WASM_BIGINT:
+    default_min_browser_version(user_settings, "Safari", 150000)
```

Comment:
Looking in this file, we seem to use single quotes 10x more often. We should probably be more consistent, but please use the more common form. In particular, on line 396 right above this diff segment we have a single quote, and its nice to not see code skipping between the two.

Reply:
Any reason you don't use black?

Reply:
I think we considered it a one point... I can't remember why we didn't go with it.  For now we just use `flake8`.

In any case single quotes in python are preferred.